cloud/deploys/staging_azure/variables.tf

```diff
@@ -18,14 +18,9 @@ variable "application_name" {
   description = "Azure Web App Name"
 }
 
-variable "postgres_admin_password" {
-  type        = string
-  description = "Postgres admin password"
-}
-
-variable "app_secret_key" {
-  type        = string
-  description = "Secret Key For the app"
+variable "key_vault_name" {
```

Comment:
do we want to set a default here

Reply:
I'm not sure, actually. What's your opinion? This needs to be manually configured so it might be easier to copy paste the key vault name from the Azure portal. Otherwise it might be named incorrectly or something.

Reply:
I could add a shell script to set up the key vault.

Reply:
i think what you did works!